modules/psap-special-resource-operator-using-configmaps.adoc

```diff
@@ -0,0 +1,309 @@
+----
++
+[source, terminal]
+----
+$ oc create cm simple-kmod-chart --from-file=cm/index.yaml --from-file=cm/simple-kmod-0.0.1.tgz -n simple-kmod
+----
+
+. Use the following `SpecialResource` manifest to deploy the simple-kmod object using the Helm chart that you created in the ConfigMap. Save this YAML as `simple-kmod-configmap.yaml`:
++
+[source,yaml]
+----
+apiVersion: sro.openshift.io/v1beta1
+kind: SpecialResource
+metadata:
+  name: simple-kmod
+spec:
+  #debug: true <1>
+  namespace: simple-kmod
+  chart:
+    name: simple-kmod
+    version: 0.0.1
+    repository:
+      name: example
+      url: cm://simple-kmod/simple-kmod-chart <2>
+  set:
+    kind: Values
+    apiVersion: sro.openshift.io/v1beta1
+    kmodNames: ["simple-kmod", "simple-procfs-kmod"]
+    buildArgs:
+    - name: "KMODVER"
+      value: "SRO"
+  driverContainer:
+    source:
+      git:
+        ref: "master"
+        uri: "https://github.com/openshift-psap/kvc-simple-kmod.git"
+----
+
+<1> Optional: Uncomment the "#debug: true" line, to have the YAML files in the chart printed in full in the Operator logs and to verify that the logs are created and templated properly.
+<2> The `spec.chart.repository.url` field tells SRO to look for the chart in a ConfigMap.
+
+. From a command line, create the `SpecialResource` file:
++
+[source,terminal]
+----
+$ oc create -f simple-kmod-configmap.yaml
+----
+
+. The `simple-kmod` resources are deployed in the `simple-kmod` namespace as specified in the object manifest. After a short time, the build pod for the `simple-kmod` driver container starts running. The build completes after a few minutes, and then the driver container pods start running. 
++
+Use `oc get pods` command to display the status of the build pods:
++
+[source,terminal]
+----
+$ oc get pods -n simple-kmod
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                                  READY   STATUS      RESTARTS   AGE
+simple-kmod-driver-build-12813789169ac0ee-1-build     0/1     Completed   0          7m12s
+simple-kmod-driver-container-12813789169ac0ee-mjsnh   1/1     Running     0          8m2s
+simple-kmod-driver-container-12813789169ac0ee-qtkff   1/1     Running     0          8m2s
+----
+
+. Use the `oc logs` command, along with the build pod name obtained from the `oc get pods` command above, to display the logs of the simple-kmod driver container image build:
++
+[source,terminal]
+----
+$ oc logs pod/simple-kmod-driver-build-12813789169ac0ee-1-build -n simple-kmod
+----
+
+. To verify that the simple-kmod kernel modules are loaded, execute the `lsmod` command in one of the driver container pods that was returned from the `oc get pods` command above. 
++
+[source,terminal]
+----
+$ oc exec -n simple-kmod -it pod/simple-kmod-driver-container-12813789169ac0ee-mjsnh -- lsmod | grep simple
+----
++
+.Example output
+[source,terminal]
+----
+simple_procfs_kmod     16384  0
+simple_kmod            16384  0
+----
+[NOTE]
```

Comment:
Add an enter between line 305 and 306 so the note displays properly. If you want to keep it together with the Example output, then add a + before the NOTE.